src/main/java/carpet/commands/TickCommand.java

```diff
@@ -35,7 +35,7 @@ public static void register(CommandDispatcher<ServerCommandSource> dispatcher)
                                 executes((c) -> setTps(c.getSource(), getFloat(c, "rate"))))).
                 then(literal("warp").
                         executes( (c)-> setWarp(c.getSource(), 0, null)).
-                        then(argument("ticks", integer(0,4000000)).
+                        then(argument("ticks", integer(0,Integer.MAX_VALUE)).
```

Comment:
you technically only need to specify the min
```suggestion
                        then(argument("ticks", integer(0)).
```